generator/templates/Default/src/App.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-    <div>
+    <v-app>
```

Comment:
```suggestion
    <v-app app-data>
```

Reply:
Wat bedoel je hier mee?

Reply:
I've looked at the Vuetify source and this doesn't do anything.

Reply:
Geen idee wat @martn001 hier bedoelde. @martn001 als jij het eens bent zet je hem dan op resolved?

Reply:
Ik dacht dat dit een standaard dingetje was voor `v-app`, dat hierdoor vuetify werd ingeladen. Maar volgensmij klopt daar niks van 😓 dus vergeet het vorig bericht...